README.md

```diff
@@ -34,17 +34,17 @@ devtools::install_github("gvelasq/tidytab")
 library(tidytab)
 
 # one-way table of frequencies
-mtcars %>% tab(cyl)
+mtcars |> tab(cyl)
 #>         cyl │      Freq.     Percent        Cum. 
 #> ────────────┼───────────────────────────────────
-#>           4 │         11        34.4        34.4 
-#>           6 │          7        21.9        56.3 
-#>           8 │         14        43.8       100.0 
+#>           4 │         11       34.38       34.38 
```

Comment:
I'd like to keep the original design where tidytab shows single decimals. The tabulations should remain the same in these examples. Could you please reknit the README? You might need to reinstall from the main branch.